src/pages/Members/Members.tsx

```diff
@@ -1,37 +1,62 @@
+import { useEffect, useState } from 'react';
+
 import { Layout } from '../../components/Layout/Layout';
 import girlHoldingBooks from '../../assets/images/girl_holding_books.png';
-import members from '../../data/members.json';
+import { getMembers } from '../../api/membersApi';
+import { MemberType } from '../../model/Member';
 import style from './Members.module.scss';
 
-export const Members: React.FC = () => (
-  <Layout>
-    <section className={style.section}>
-      <div className={style.sectionHeader}>
-        <div className={style.sectionLeft}>
-          <h1 className={style.underlined}>our great members</h1>
-          <p className={style.description}>
-            Our community is made up of book lovers from all walks of life. We
-            have students, teachers, parents, and professionals who are
-            passionate about reading and sharing their thoughts with others.
-            Join us today and meet new friends who share your love for reading.
-          </p>
-        </div>
-        <div className={style.sectionRight}>
-          <img className={style.illustration} src={girlHoldingBooks} alt="" />
+export const Members: React.FC = () => {
+  const [members, setMembers] = useState<MemberType[]>([]);
+
+  useEffect(() => {
+    const getData = async () => {
+      const data = await getMembers();
+      setMembers(data);
+    };
+
+    getData();
+  }, []);
+
+  const formatName = (name: string, surname: string) => {
+    const surnameInitial = surname ? `${surname.charAt(0)}.` : '';
```

Comment:
surname is required, it will always be present, I believe this can simply be `{name}{' '}{surname.charAt())` in the JSX. Btw leaving out the 0 will use 0.

I do like abstraction like this! But I believe in this case we don't need it